ocrd_utils/ocrd_utils/__init__.py

```diff
@@ -301,27 +356,42 @@ def get_local_filename(url, start=None):
         url = url[len(start):]
     return url
 
-
-def image_from_polygon(image, polygon):
+def image_from_polygon(image, polygon, fill='background', transparency=False):
     """"Mask an image with a polygon.
 
     Given a PIL.Image ``image`` and a numpy array ``polygon``
-    of relative coordinates into the image, put everything
-    outside the polygon hull to the background. Since ``image``
-    is not necessarily binarized yet, determine the background
-    from the median color (instead of white).
-
+    of relative coordinates into the image, fill everything
+    outside the polygon hull to a color according to ``fill``:
+    - if ``background`` (the default),
+      then use the median color of the image;
+    - otherwise use the given color, e.g. ``white`` or (255,255,255).
+    Moreover, if ``transparency`` is true, then add an alpha channel
+    from the polygon mask (i.e. everything outside the polygon will
+    be transparent for those that can interpret alpha channels).
+    (Images which already have an alpha channel will have them
+    shrinked from the polygon mask.)
```

Comment:
will have **them** (?) *expanded to* the polygon mask?

Reply:
yes, _expanded_ (because alpha channels actually encode opacity, the reverse of transparency). I re-formulated to clarify.